core/src/main/groovy/noe/tomcat/configure/SecureHttpConnectorTomcat.groovy

```diff
@@ -78,39 +116,82 @@ public class SecureHttpConnectorTomcat extends ConnectorTomcatAbstract<SecureHtt
     return this.sslPassword
   }
 
+  public String getSslImplementationName() {
+    return this.sslImplementationName
+  }
+
+  public Boolean getSslEnabledProtocols() {
+    return this.sslEnabledProtocols
+  }
+
   public SecureHttpConnectorTomcat setSslProtocol(String sslProtocol) {
     this.sslProtocol = sslProtocol
     return this
   }
 
-  public SecureHttpConnectorTomcat setKeystoreFile(String keystoreFile) {
+  public SecureHttpConnectorTomcat setKeystoreFile(String keystoreFile = keystoreFilePath) {
     this.keystoreFile = keystoreFile
     return this
   }
 
-  public SecureHttpConnectorTomcat setKeystorePass(String keystorePass) {
+  public SecureHttpConnectorTomcat setKeystorePass(String keystorePass = password) {
     this.keystorePass = keystorePass
     return this
   }
 
+  public SecureHttpConnectorTomcat setKeystoreType(String keystoreType) {
+    this.keystoreType = keystoreType
+    return this
+  }
+
+  public SecureHttpConnectorTomcat setTruststoreFile(String truststoreFile) {
+    this.truststoreFile = truststoreFile
+    return this
+  }
+
+  public SecureHttpConnectorTomcat setTruststorePass(String truststorePass) {
+    this.truststorePass = truststorePass
+    return this
+  }
+
+  public SecureHttpConnectorTomcat setTruststoreType(String truststoreType) {
+    this.truststoreType = truststoreType
+    return this
+  }
+
   public SecureHttpConnectorTomcat setClientAuth(boolean clientAuth) {
     this.clientAuth = clientAuth
     return this
   }
 
-  public SecureHttpConnectorTomcat setSslCertificateFile(String sslCertificateFile) {
+  public SecureHttpConnectorTomcat setSslCertificateFile(String sslCertificateFile = sslCertificate) {
     this.sslCertificateFile = sslCertificateFile
     return this
   }
 
-  public SecureHttpConnectorTomcat setSslCertificateKeyFile(String sslCertificateKeyFile) {
+  public SecureHttpConnectorTomcat setSslCACertificateFile(String sslCACertificateFile) {
```

Comment:
are we 100% sure that this method is not used anywhere? Shouldn't we deprecate it 1st?

Reply:
I did not remove this.It's right below the comment.My goal here was just the addition of setSslCACertificateFile setter

Reply:
I see. Thank you for explanation.